tests/integration_tests/performance/test_network_tcp_throughput.py

```diff
@@ -303,7 +303,11 @@ def pipes(basevm, host_ip, current_avail_cpu, env_id):
 @pytest.mark.nonci
 @pytest.mark.timeout(3600)
 def test_network_tcp_throughput(bin_cloner_path, results_file_dumper):
-    """Test network throughput driver for multiple artifacts."""
+    """
+    Test network throughput for multiple artifacts.
```

Comment:
Instead of "artifacts" I d say "for multiple microVM configurations"

Reply:
done